ccl/testccl/workload/schemachange: TestWorkload failed

[(ccl/testccl/workload/schemachange).TestWorkload failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2806895&tab=buildLog) on [master@ab7362161d5e3b88f4730db94b95c5926b5eb3af](https://github.com/cockroachdb/cockroach/commits/ab7362161d5e3b88f4730db94b95c5926b5eb3af):

```
1, 29, rangelog, 13
1, 29, replication_constraint_stats, 25
1, 29, replication_critical_localities, 26
1, 29, replication_stats, 27
1, 29, reports_meta, 28
1, 29, role_members, 23
1, 29, role_options, 33
1, 29, scheduled_jobs, 37
1, 29, settings, 6
1, 29, sqlliveness, 39
1, 29, statement_bundle_chunks, 34
1, 29, statement_diagnostics, 36
1, 29, statement_diagnostics_requests, 35
1, 29, table_statistics, 20
1, 29, tenants, 8
1, 29, ui, 14
1, 29, users, 4
1, 29, web_sessions, 19
1, 29, zones, 5
50, 0, public, 29
51, 0, public, 29
52, 0, public, 29
52, 0, schema21, 62
52, 29, _crdb_internal_region, 64
52, 29, crdb_internal_region, 63
I210323 22:26:12.630571 10325 9@util/log/event_log.go:32  [n1,client=127.0.0.1:40976,hostssl,user=root,intExec=backup-lookup-tenant] 3159 ={"Timestamp":1616538372600117161,"EventType":"query_execute","Statement":"SELECT id, active, info FROM \"\".system.tenants","User":"root","ApplicationName":"$ internal-backup-lookup-tenant","ExecMode":"exec-internal","Age":8.822355,"FullTableScan":true}
I210323 22:26:12.642492 10326 9@util/log/event_log.go:32  [n1,client=127.0.0.1:40976,hostssl,user=root,intExec=job-row-insert] 3160 ={"Timestamp":1616538372632047538,"EventType":"query_execute","Statement":"INSERT INTO \"\".system.jobs(id, status, payload, progress, claim_session_id, claim_instance_id) VALUES ($1, $2, $3, $4, $5, $6)","User":"root","ApplicationName":"$ internal-job-row-insert","PlaceholderValues":["643786252720635905","'running'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f743204343e3f4052240a00120a08dcb2cdc5dca2c6b7161a146e6f64656c6f63616c3a2f2f302f6261636b7570'","'\\x10f0dbdddbbac7ef025200'","'\\x16ba01fba0324fdeb8c6f38ffbc5a1ae'","1"],"ExecMode":"exec-internal","NumRows":1,"Age":10.106648}
I210323 22:26:12.684922 10349 9@util/log/event_log.go:32  [n1,client=127.0.0.1:40976,hostssl,user=root,intExec=log-job] 3161 ={"Timestamp":1616538372671621823,"EventType":"query_execute","Statement":"SELECT status, payload, progress, claim_session_id FROM \"\".system.jobs WHERE id = $1 FOR UPDATE","User":"root","ApplicationName":"$ internal-log-job","PlaceholderValues":["643786252720635905"],"ExecMode":"exec-internal","NumRows":1,"Age":13.052351}
I210323 22:26:12.700214 10351 9@util/log/event_log.go:32  [n1,client=127.0.0.1:40976,hostssl,user=root,intExec=job-update] 3162 ={"Timestamp":1616538372671621823,"EventType":"query_execute","Statement":"UPDATE \"\".system.jobs SET status = $2, payload = $3 WHERE id = $1","User":"root","ApplicationName":"$ internal-job-update","PlaceholderValues":["643786252720635905","'running'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f7418b983e1dbbac7ef023204343e3f4052240a00120a08dcb2cdc5dca2c6b7161a146e6f64656c6f63616c3a2f2f302f6261636b7570'"],"ExecMode":"exec-internal","NumRows":1,"Age":13.632049}
I210323 22:26:12.709359 10251 jobs/registry.go:1175  [n1] 3163  BACKUP job 643786252720635905: stepping through state running with error: <nil>
I210323 22:26:12.758856 10356 9@util/log/event_log.go:32  [n1,job=643786252720635905,intExec=lookup-schedule-info] 3164 ={"Timestamp":1616538372745548302,"EventType":"query_execute","Statement":"SELECT created_by_id FROM \"\".system.jobs WHERE (id = $1) AND (created_by_type = $2)","User":"node","ApplicationName":"$ internal-lookup-schedule-info","PlaceholderValues":["643786252720635905","'crdb_schedule'"],"ExecMode":"exec-internal","Age":13.064578}
I210323 22:26:12.760043 10251 jobs/registry.go:1175  [n1] 3165  BACKUP job 643786252720635905: stepping through state succeeded with error: <nil>
I210323 22:26:12.773568 10357 9@util/log/event_log.go:32  [n1,intExec=log-job] 3166 ={"Timestamp":1616538372760387669,"EventType":"query_execute","Statement":"SELECT status, payload, progress, claim_session_id FROM \"\".system.jobs WHERE id = $1 FOR UPDATE","User":"root","ApplicationName":"$ internal-log-job","PlaceholderValues":["643786252720635905"],"ExecMode":"exec-internal","NumRows":1,"Age":12.981474}
I210323 22:26:12.790549 10359 9@util/log/event_log.go:32  [n1,intExec=job-update] 3167 ={"Timestamp":1616538372760387669,"EventType":"query_execute","Statement":"UPDATE \"\".system.jobs SET status = $2, payload = $3, progress = $4 WHERE id = $1","User":"root","ApplicationName":"$ internal-job-update","PlaceholderValues":["643786252720635905","'succeeded'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f7418b983e1dbbac7ef022080b8e6dbbac7ef023204343e3f4052240a00120a08dcb2cdc5dca2c6b7161a146e6f64656c6f63616c3a2f2f302f6261636b7570'","'\\x0d0000803f10c4c6e5dbbac7ef025200'"],"ExecMode":"exec-internal","NumRows":1,"Age":15.298867}
I210323 22:26:12.801368 6349 9@util/log/event_log.go:32  [n1,client=127.0.0.1:40976,hostssl,user=root] 3168 ={"Timestamp":1616538372600117161,"EventType":"query_execute","Statement":"BACKUP DATABASE schemachange TO 'nodelocal://0/backup'","User":"root","ExecMode":"exec","NumRows":1,"Age":201.52194,"TxnCounter":7}
    schema_change_external_test.go:81: backup in /go/src/github.com/cockroachdb/cockroach/artifacts/TestWorkload177599007
I210323 22:26:12.802946 13 testutils/testcluster/testcluster.go:101  [-] 3169  TestCluster quiescing nodes
E210323 22:26:12.804834 10286 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 3170  making txn commit explicit failed for "unnamed" meta={id=6e1198b5 pri=0.00438211 epo=0 ts=1616538372.760387669,0 min=1616538372.760387669,0 seq=6} lock=true stat=STAGING rts=1616538372.760387669,0 wto=false gul=1616538373.260387669,0 int=1 ifw=5: result is ambiguous (server shutdown)
W210323 22:26:12.807100 10354 kv/kvserver/intentresolver/intent_resolver.go:758  [-] 3171  failed to gc transaction record: could not GC completed transaction anchored at /Table/15/1/643786252720635905: node unavailable; try another peer
W210323 22:26:12.808725 1462 sql/sqlliveness/slinstance/slinstance.go:183  [n2] 3172  exiting heartbeat loop
W210323 22:26:12.811784 1463 jobs/registry.go:729  [-] 3173  canceling all adopted jobs due to stopper quiescing
W210323 22:26:12.814841 1801 jobs/registry.go:729  [-] 3174  canceling all adopted jobs due to stopper quiescing
W210323 22:26:12.816624 1970 kv/kvserver/closedts/sidetransport/receiver.go:125  [n2] 3175  closed timestamps side-transport connection dropped from node: 1
W210323 22:26:12.818102 365 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 3176  exiting heartbeat loop
W210323 22:26:12.818882 1800 sql/sqlliveness/slinstance/slinstance.go:183  [n3] 3177  exiting heartbeat loop
W210323 22:26:12.823340 366 jobs/registry.go:729  [-] 3178  canceling all adopted jobs due to stopper quiescing
W210323 22:26:12.823975 1195 2@gossip/gossip.go:1491  [n2] 3179  no incoming or outgoing connections
W210323 22:26:12.831381 2137 kv/kvserver/closedts/sidetransport/receiver.go:125  [n3] 3180  closed timestamps side-transport connection dropped from node: 1
I210323 22:26:12.881459 314 kv/kvclient/kvcoord/transport_race.go:106  [-] 3181  transport race promotion: ran 65 iterations on up to 3625 requests
--- FAIL: TestWorkload (19.66s)

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestWorkload PKG=./pkg/ccl/testccl/workload/schemachange TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestWorkload.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(ccl/testccl/workload/schemachange).TestWorkload failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2808169&tab=buildLog) on [master@9fa4b125bfb07552b43ba4fd52c9301afd7a937b](https://github.com/cockroachdb/cockroach/commits/9fa4b125bfb07552b43ba4fd52c9301afd7a937b):

```
1, 29, zones, 5
50, 0, public, 29
51, 0, public, 29
52, 0, public, 29
52, 0, schema1, 53
52, 0, schema24, 61
52, 29, _crdb_internal_region, 58
52, 29, _enum21, 60
52, 29, crdb_internal_region, 57
52, 29, enum21, 59
52, 29, seq20, 56
I210324 05:29:14.510911 8485 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=backup-lookup-tenant] 3439 ={"Timestamp":1616563754506777271,"EventType":"query_execute","Statement":"SELECT id, active, info FROM \"\".system.tenants","User":"root","ApplicationName":"$ internal-backup-lookup-tenant","ExecMode":"exec-internal","Age":1.069421,"FullTableScan":true}
I210324 05:29:14.512503 8486 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=job-row-insert] 3440 ={"Timestamp":1616563754511158623,"EventType":"query_execute","Statement":"INSERT INTO \"\".system.jobs(id, status, payload, progress, claim_session_id, claim_instance_id) VALUES ($1, $2, $3, $4, $5, $6)","User":"root","ApplicationName":"$ internal-job-row-insert","PlaceholderValues":["643869424062136321","'running'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f743208343538393a3b3c3d52360a00120a08b2fda4c1b785ccb7161a146e6f64656c6f63616c3a2f2f302f6261636b75703a1052a4851b7f0b4480a722eb7cdf1042c9'","'\\x10b7d6e0a299c8ef025200'","'\\x6eff0489203449a28c6d5dcd8a29e756'","1"],"ExecMode":"exec-internal","NumRows":1,"Age":1.293313}
I210324 05:29:14.524752 8488 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=protectedts-protect] 3441 ={"Timestamp":1616563754511158623,"EventType":"query_execute","Statement":"WITH current_meta AS (SELECT version, num_records, num_spans, total_bytes FROM \"\".system.protected_ts_meta UNION ALL SELECT 0 AS version, 0 AS num_records, 0 AS num_spans, 0 AS total_bytes ORDER BY version DESC LIMIT 1), checks AS (SELECT new_version, new_num_records, new_num_spans, new_total_bytes, (((($1 > 0) AND (new_num_spans > $1)) OR (($2 > 0) AND (new_total_bytes > $2))) OR EXISTS (SELECT * FROM \"\".system.protected_ts_records WHERE id = $4)) AS failed FROM (SELECT version + 1 AS new_version, num_records + 1 AS new_num_records, num_spans + $3 AS new_num_spans, ((total_bytes + length($9)) + length($6)) + COALESCE(length($7), 0) AS new_total_bytes FROM \"\".\"\".current_meta)), updated_meta AS (UPSERT INTO \"\".system.protected_ts_meta(version, num_records, num_spans, total_bytes) (SELECT new_version, new_num_records, new_num_spans, new_total_bytes FROM \"\".\"\".checks WHERE NOT failed) RETURNING version, num_records, num_spans,
I210324 05:29:14.524752 8488 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=protectedts-protect] 3441 | total_bytes), new_record AS (INSERT INTO \"\".system.protected_ts_records(id, ts, meta_type, meta, num_spans, spans) (SELECT $4, $5, $6, $7, $8, $9 WHERE NOT EXISTS (SELECT * FROM \"\".\"\".checks WHERE failed)) RETURNING id) SELECT failed, num_spans AS prev_spans, total_bytes AS prev_total_bytes, version AS prev_version FROM \"\".\"\".checks, \"\".\"\".current_meta","User":"node","ApplicationName":"$ internal-protectedts-protect","PlaceholderValues":["32768","1048576","1","'\\x52a4851b7f0b4480a722eb7cdf1042c9'","'1616563754506993330.0000000000'","'jobs'","'\\x363433383639343234303632313336333231'","1","'\\x3a081a02c0892202c08a'"],"ExecMode":"exec-internal","NumRows":1,"Age":5.520428,"FullTableScan":true}
I210324 05:29:14.527088 8461 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=log-job] 3442 ={"Timestamp":1616563754525848475,"EventType":"query_execute","Statement":"SELECT status, payload, progress, claim_session_id FROM \"\".system.jobs WHERE id = $1 FOR UPDATE","User":"root","ApplicationName":"$ internal-log-job","PlaceholderValues":["643869424062136321"],"ExecMode":"exec-internal","NumRows":1,"Age":1.214993}
I210324 05:29:14.528488 8463 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root,intExec=job-update] 3443 ={"Timestamp":1616563754525848475,"EventType":"query_execute","Statement":"UPDATE \"\".system.jobs SET status = $2, payload = $3 WHERE id = $1","User":"root","ApplicationName":"$ internal-job-update","PlaceholderValues":["643869424062136321","'running'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f7418ced3e1a299c8ef023208343538393a3b3c3d52360a00120a08b2fda4c1b785ccb7161a146e6f64656c6f63616c3a2f2f302f6261636b75703a1052a4851b7f0b4480a722eb7cdf1042c9'"],"ExecMode":"exec-internal","NumRows":1,"Age":1.267665}
I210324 05:29:14.529217 8491 jobs/registry.go:1175  [n1] 3444  BACKUP job 643869424062136321: stepping through state running with error: <nil>
I210324 05:29:14.532859 8492 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=protectedts-GetRecord] 3445 ={"Timestamp":1616563754529326315,"EventType":"query_execute","Statement":"SELECT id, ts, meta_type, meta, spans, verified FROM \"\".system.protected_ts_records WHERE id = $1","User":"node","ApplicationName":"$ internal-protectedts-GetRecord","PlaceholderValues":["'\\x52a4851b7f0b4480a722eb7cdf1042c9'"],"ExecMode":"exec-internal","NumRows":1,"Age":3.505504}
I210324 05:29:14.536374 8044 9@util/log/event_log.go:32  [intExec=protectedts-GetMetadata] 3446 ={"Timestamp":1616563754533512173,"EventType":"query_execute","Statement":"WITH current_meta AS (SELECT version, num_records, num_spans, total_bytes FROM \"\".system.protected_ts_meta UNION ALL SELECT 0 AS version, 0 AS num_records, 0 AS num_spans, 0 AS total_bytes ORDER BY version DESC LIMIT 1) SELECT version, num_records, num_spans, total_bytes FROM \"\".\"\".current_meta","User":"node","ApplicationName":"$ internal-protectedts-GetMetadata","ExecMode":"exec-internal","NumRows":1,"Age":2.822959,"FullTableScan":true}
I210324 05:29:14.538703 8412 9@util/log/event_log.go:32  [intExec=protectedts-GetMetadata] 3447 ={"Timestamp":1616563754533512173,"EventType":"query_execute","Statement":"WITH current_meta AS (SELECT version, num_records, num_spans, total_bytes FROM \"\".system.protected_ts_meta UNION ALL SELECT 0 AS version, 0 AS num_records, 0 AS num_spans, 0 AS total_bytes ORDER BY version DESC LIMIT 1) SELECT version, num_records, num_spans, total_bytes FROM \"\".\"\".current_meta","User":"node","ApplicationName":"$ internal-protectedts-GetMetadata","ExecMode":"exec-internal","NumRows":1,"Age":2.202943,"FullTableScan":true}
I210324 05:29:14.540866 8506 9@util/log/event_log.go:32  [intExec=protectedts-GetRecords] 3448 ={"Timestamp":1616563754533512173,"EventType":"query_execute","Statement":"SELECT id, ts, meta_type, meta, spans, verified FROM \"\".system.protected_ts_records","User":"node","ApplicationName":"$ internal-protectedts-GetRecords","ExecMode":"exec-internal","NumRows":1,"Age":2.053321,"FullTableScan":true}
I210324 05:29:14.542735 8416 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=protectedts-MarkVerified] 3449 ={"Timestamp":1616563754541109677,"EventType":"query_execute","Statement":"UPDATE \"\".system.protected_ts_records SET verified = true WHERE id = $1 RETURNING true","User":"node","ApplicationName":"$ internal-protectedts-MarkVerified","PlaceholderValues":["'\\x52a4851b7f0b4480a722eb7cdf1042c9'"],"ExecMode":"exec-internal","NumRows":1,"Age":1.603008}
I210324 05:29:14.551100 8557 ccl/backupccl/backup_processor.go:315  [n3] 3450  sending ExportRequest for span /Table/56/{1-2} (attempt 1, priority normal)
I210324 05:29:14.552270 8557 ccl/storageccl/export.go:98  [n3,s3,r37/3:/{Table/56-Max}] 3451  export [/Table/56/1,/Table/56/2)
I210324 05:29:14.557810 8561 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=log-job] 3452 ={"Timestamp":1616563754555547450,"EventType":"query_execute","Statement":"SELECT status, payload, progress, claim_session_id FROM \"\".system.jobs WHERE id = $1 FOR UPDATE","User":"root","ApplicationName":"$ internal-log-job","PlaceholderValues":["643869424062136321"],"ExecMode":"exec-internal","NumRows":1,"Age":2.195696}
I210324 05:29:14.559850 8563 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=job-update] 3453 ={"Timestamp":1616563754555547450,"EventType":"query_execute","Statement":"UPDATE \"\".system.jobs SET progress = $2 WHERE id = $1","User":"root","ApplicationName":"$ internal-job-update","PlaceholderValues":["643869424062136321","'\\x0d0000803f109bb1e3a299c8ef025200'"],"ExecMode":"exec-internal","NumRows":1,"Age":1.895271}
I210324 05:29:14.567419 8472 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=get-table-statistics] 3454 ={"Timestamp":1616563754566916015,"EventType":"query_execute","Statement":"SELECT \"tableID\", \"statisticID\", name, \"columnIDs\", \"createdAt\", \"rowCount\", \"distinctCount\", \"nullCount\", histogram FROM \"\".system.table_statistics WHERE \"tableID\" = $1 ORDER BY \"createdAt\" DESC","User":"root","ApplicationName":"$ internal-get-table-statistics","PlaceholderValues":["56"],"ExecMode":"exec-internal","Age":1.425832,"TxnCounter":1}
I210324 05:29:14.574784 8473 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=protectedts-Release] 3455 ={"Timestamp":1616563754569521291,"EventType":"query_execute","Statement":"WITH current_meta AS (SELECT version, num_records, num_spans, total_bytes FROM \"\".system.protected_ts_meta UNION ALL SELECT 0 AS version, 0 AS num_records, 0 AS num_spans, 0 AS total_bytes ORDER BY version DESC LIMIT 1), record AS (SELECT id, num_spans AS record_spans, (length(spans) + length(meta_type)) + COALESCE(length(meta), 0) AS record_bytes FROM \"\".system.protected_ts_records WHERE id = $1), updated_meta AS (UPSERT INTO \"\".system.protected_ts_meta(version, num_records, num_spans, total_bytes) (SELECT version, num_records, num_spans, total_bytes FROM (SELECT version + 1 AS version, num_records - 1 AS num_records, num_spans - record_spans AS num_spans, total_bytes - record_bytes AS total_bytes FROM \"\".\"\".current_meta RIGHT JOIN \"\".\"\".record ON true)) RETURNING 1) DELETE FROM \"\".system.protected_ts_records AS r WHERE EXISTS (SELECT NULL FROM \"\".\"\".record WHERE r.id = record.id) RETURNING NULL","User":"node","Appli
I210324 05:29:14.574784 8473 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=protectedts-Release] 3455 |cationName":"$ internal-protectedts-Release","PlaceholderValues":["'\\x52a4851b7f0b4480a722eb7cdf1042c9'"],"ExecMode":"exec-internal","NumRows":1,"Age":5.21653,"FullTableScan":true}
I210324 05:29:14.581239 8566 9@util/log/event_log.go:32  [n1,job=643869424062136321,intExec=lookup-schedule-info] 3456 ={"Timestamp":1616563754579790750,"EventType":"query_execute","Statement":"SELECT created_by_id FROM \"\".system.jobs WHERE (id = $1) AND (created_by_type = $2)","User":"node","ApplicationName":"$ internal-lookup-schedule-info","PlaceholderValues":["643869424062136321","'crdb_schedule'"],"ExecMode":"exec-internal","Age":1.421644}
I210324 05:29:14.581323 8491 jobs/registry.go:1175  [n1] 3457  BACKUP job 643869424062136321: stepping through state succeeded with error: <nil>
I210324 05:29:14.582649 8567 9@util/log/event_log.go:32  [n1,intExec=log-job] 3458 ={"Timestamp":1616563754581350565,"EventType":"query_execute","Statement":"SELECT status, payload, progress, claim_session_id FROM \"\".system.jobs WHERE id = $1 FOR UPDATE","User":"root","ApplicationName":"$ internal-log-job","PlaceholderValues":["643869424062136321"],"ExecMode":"exec-internal","NumRows":1,"Age":1.276712}
I210324 05:29:14.584220 8569 9@util/log/event_log.go:32  [n1,intExec=job-update] 3459 ={"Timestamp":1616563754581350565,"EventType":"query_execute","Statement":"UPDATE \"\".system.jobs SET status = $2, payload = $3, progress = $4 WHERE id = $1","User":"root","ApplicationName":"$ internal-job-update","PlaceholderValues":["643869424062136321","'succeeded'","'\\x0a364241434b555020444154414241534520736368656d616368616e676520544f20276e6f64656c6f63616c3a2f2f302f6261636b7570271204726f6f7418ced3e1a299c8ef0220e085e5a299c8ef023208343538393a3b3c3d52360a00120a08b2fda4c1b785ccb7161a146e6f64656c6f63616c3a2f2f302f6261636b75703a1052a4851b7f0b4480a722eb7cdf1042c9'","'\\x0d0000803f10e7fae4a299c8ef025200'"],"ExecMode":"exec-internal","NumRows":1,"Age":1.426557}
I210324 05:29:14.586236 4711 9@util/log/event_log.go:32  [n1,client=127.0.0.1:35626,hostssl,user=root] 3460 ={"Timestamp":1616563754506777271,"EventType":"query_execute","Statement":"BACKUP DATABASE schemachange TO 'nodelocal://0/backup'","User":"root","ExecMode":"exec","NumRows":1,"Age":79.493515,"TxnCounter":7}
    schema_change_external_test.go:81: backup in /go/src/github.com/cockroachdb/cockroach/artifacts/TestWorkload083539255
I210324 05:29:14.586478 14 testutils/testcluster/testcluster.go:101  [-] 3461  TestCluster quiescing nodes
W210324 05:29:14.586593 74 sql/sqlliveness/slinstance/slinstance.go:183  [n1] 3462  exiting heartbeat loop
E210324 05:29:14.586606 8507 kv/kvclient/kvcoord/txn_interceptor_committer.go:452  [-] 3463  making txn commit explicit failed for "unnamed" meta={id=64df2e69 pri=0.00808467 epo=0 ts=1616563754.581350565,0 min=1616563754.581350565,0 seq=6} lock=true stat=STAGING rts=1616563754.581350565,0 wto=false gul=1616563755.081350565,0 int=1 ifw=5: result is ambiguous (server shutdown)
W210324 05:29:14.586740 6559 kv/kvserver/closedts/sidetransport/receiver.go:125  [n2] 3464  closed timestamps side-transport connection dropped from node: 3
W210324 05:29:14.586741 8480 kv/kvserver/intentresolver/intent_resolver.go:758  [-] 3465  failed to gc transaction record: could not GC completed transaction anchored at /Table/31/1/1/0: node unavailable; try another peer
W210324 05:29:14.587011 1822 jobs/registry.go:729  [-] 3466  canceling all adopted jobs due to stopper quiescing
W210324 05:29:14.587033 6626 kv/kvserver/closedts/sidetransport/receiver.go:125  [n1] 3467  closed timestamps side-transport connection dropped from node: 3
W210324 05:29:14.587152 75 jobs/registry.go:729  [-] 3468  canceling all adopted jobs due to stopper quiescing
W210324 05:29:14.587277 1882 kv/kvserver/closedts/sidetransport/receiver.go:125  [n3] 3469  closed timestamps side-transport connection dropped from node: 1
W210324 05:29:14.587324 1821 sql/sqlliveness/slinstance/slinstance.go:183  [n3] 3470  exiting heartbeat loop
W210324 05:29:14.587528 2373 kv/kvserver/closedts/sidetransport/receiver.go:125  [n2] 3471  closed timestamps side-transport connection dropped from node: 1
W210324 05:29:14.587605 1425 jobs/registry.go:729  [-] 3472  canceling all adopted jobs due to stopper quiescing
W210324 05:29:14.587841 1424 sql/sqlliveness/slinstance/slinstance.go:183  [n2] 3473  exiting heartbeat loop
--- FAIL: TestWorkload (3.83s)

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestWorkload PKG=./pkg/ccl/testccl/workload/schemachange TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestWorkload.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/workload/schemachange/operation_generator.go
+++ b/pkg/workload/schemachange/operation_generator.go
@@ -184,7 +184,7 @@ var opWeights = []int{
 	addColumn:               1,
 	addConstraint:           0, // TODO(spaskob): unimplemented
 	addForeignKeyConstraint: 0,
-	addRegion:               1,
+	addRegion:               0, // TODO(otan): re-enable after fixing SHOW REGIONS FROM DATABASE
 	addUniqueConstraint:     0,
 	createIndex:             1,
 	createSequence:          1,

